apps/api/src/.env.development

```diff
@@ -22,3 +22,5 @@ NEW_RELIC_APP_NAME="[DEV] - api"
 
 VERCEL_REDIRECT_URI=https://dev.web.novu.co/auth/login
 VERCEL_BASE_URL=https://api.vercel.com
+
+FF_IS_TOPIC_NOTIFICATION_ENABLED=false
```

Comment:
While we do not have a different way to set feature flags I'd rather us to name them in a way to differentiate.